File: ws/Segarra/u1/Ws05ProgrammingLanguagues/MultiplicationTables.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package com.mycompany.multiplicationtables;

/**
 *
 * @author eduse
 */
public class MultiplicationTables {

    public static void main(String[] args) {
        for (int i = 1; i <= 10; i++) {
            System.out.println("Tabla del " + i + ":");
            // Iterar desde 1 hasta 10 para cada tabla de multiplicar
            for (int j = 1; j <= 10; j++) {
                // Imprimir el resultado de la multiplicación
                System.out.println(i + " x " + j + " = " + (i * j));
            }
            // Línea en blanco para separar las tablas
            System.out.println();
        }
    }
}

File: ws/pantoja/u1/ws05ProgrammingLenguages/multiplicationTables.py
def tablas_multiplicar():
    for i in range(1, 11):
        print(f"Tabla de multiplicar del {i}:")
        for j in range(1, 11):
            print(f"{i} x {j} = {i * j}")
        print()  # línea en blanco para separar las tablas

# Llamamos a la función para mostrar las tablas de multiplicar
tablas_multiplicar()
